patches/ui-webui-resources-cr_elements-cr_elements.gni.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/ui/webui/resources/cr_elements/cr_elements.gni b/ui/webui/resources/cr_elements/cr_elements.gni
+index 263e74b8727efe0e9a6589360333a300a9bcbdfc..43989d8a5914b36f802f720ef838e1a9be14f2d3 100644
+--- a/ui/webui/resources/cr_elements/cr_elements.gni
++++ b/ui/webui/resources/cr_elements/cr_elements.gni
+@@ -50,7 +50,7 @@ if (include_polymer) {
+     "cr_view_manager/cr_view_manager.ts",
+   ]
+ 
+-  if (is_chromeos) {
++  if (!is_android || is_chromeos) {
```

Comment:
I don't think a gn patch is the right way to handle this cc @petemill 
in the meantime you should avoid modifying lines in patches so this should be
`if (is_android) { web_component_files_polymer_ts -= [ "cr_searchable_drop_down/cr_searchable_drop_down.ts" ] }` after it gets added

Reply:
Updated to use the technique you mentioned d97b7507b9cca2f99320bd070909fbffa749ae20.

Just as an FYI, this patch is an update and simplification of a pre-existing patch here https://github.com/brave/brave-core/blob/master/patches/ui-webui-resources-cr_elements-BUILD.gn.patch.